src/__tests__/App.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import App from '../components/App';
+import configureStore from '../store/rootStore';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+
+const APP_VERSION = '0.1.0-SNAPSHOT';
+const FOOTER_VERSION_TEXT = `version: ${APP_VERSION}`;
+
+describe('App', () => {
+    it('shows right version', () => {
+        //  Mock fetch call
+        const mockSuccessResponse = { build: { version: APP_VERSION } };
+        const mockJsonPromise = Promise.resolve(mockSuccessResponse);
+        const mockFetchPromise = Promise.resolve({
+            json: () => mockJsonPromise,
+        });
+        jest.spyOn(global, 'fetch').mockImplementation(() => mockFetchPromise);
+
+        let store = configureStore();
+
+        const component = mount(
+            <Provider store={store}>
+                <Router basename="/demo">
+                    <App />
+                </Router>
+            </Provider>
+        );
+
+        //  Subscribe before 1st action - SERVER_INFO_SET
+        const unsubscribe = store.subscribe(() => {
+            unsubscribe();
+            const version = component.find('.app-footer > div').text();
+            expect(version).toEqual(FOOTER_VERSION_TEXT);
+        });
+
+        expect(global.fetch).toHaveBeenCalledTimes(1);
+        expect(global.fetch).toHaveBeenCalledWith('/api/file/status/info');
```

Comment:
After you update the code in previous commit to include `/about` page please also verify that the correct version is displayed there.
Make sure that test fails if you e.g. change the server response to `0.2.0-SNAPSHOT`, otherwise the test is useless :)